chainer/functions/loss/softmax_cross_entropy.py

```diff
@@ -243,9 +243,19 @@ class weight ``class_weight[i]`` is actually multiplied to
             value.
         ignore_label (int): Label value you want to ignore. Its default value
             is ``-1``. See description of the argument `t`.
+        reduce (str): Variable holding a ``str`` which
+            determines whether to reduce the shape of the input.
```

Comment:
As the input shape is not the target of reduction, it is better "... to reduce loss values".